### PR TITLE
cleanup(docs): adjust falco readme style and content

### PR DESCRIPTION
**Pull request by incertum**
**What type of PR is this?**

> Uncomment one (or more) `/kind <>` lines:

> /kind bug

> /kind cleanup

> /kind design

/kind documentation

> /kind failing-test

> /kind feature

> /kind release

**Any specific area of the project related to this PR?**

> Uncomment one (or more) `/area <>` lines:

> /area build

> /area engine

> /area tests

> /area proposals

> /area CI

**What this PR does / why we need it**:

Adjust falco readme style and content in order to make the information provided here more precise and relevant for adopters. Furthermore, start the transition towards the Falco website being a single consolidated source of information.

cc @falcosecurity/core-maintainers @leogr 

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
cleanup(docs): adjust falco readme style and content 
```

### CODE REVIEW

**Review by leogr — COMMENTED** [review]
Hey @incertum 

 I can only thank you for this PR :star_struck:  

This is a great start to finally getting consistent information in the README.

So, willing to help, I was as exhaustive as possible in my suggestions. Some suggestions, however, are only minor aspects. Let me know what you think.
Anyway, overall, it SGTM!

File: README.md [hunk 1]
```diff
@@ -58,107 +54,77 @@ Notes:
...
 
-Falco is designed to be extensible such that it can be built into cloud-native applications and infrastructure.
+For more information, visit the official hub of the Falco ecosystem: [falcosecurity/evolution](https://github.com/falcosecurity/evolution). It provides valuable insights and information about the project's repositories.
 
-Falco has a [gRPC](https://falco.org/docs/grpc/) endpoint and an API defined in [protobuf](https://github.com/falcosecurity/falco/blob/master/userspace/falco/outputs.proto).
-The Falco Project supports various SDKs for this endpoint.
+## Getting Started with Falco
 
-### SDKs
+Carefully review and follow the [official guide and documentation](https://falco.org/docs/getting-started/). Integration timelines may vary based on your organization's needs, typically taking several weeks of ramp-up time.
```
[the remark this answers is not in the document]
Comment (incertum) [reply README.md]: 🤦‍♀️ agree, but yeah it's a reflection of reality check ...

**Review by leogr — APPROVED** [review]
LGTM, now.

Thank you :pray:

**Comment by leogr** [comment]
> Also will we have one separate binary for modern bpf or just one binary that now always included modern bpf? 

No separate binary.  It is always included.

> Added links to the artifacts repo containing the stable default drivers may be beneficial too?

Not sure, perhaps better in a follow-up PR.

**Review by jasondellaluce — APPROVED** [review]
/approve

**Pull request merged**